Src/Base/AMReX_GpuLaunch.H

```diff
@@ -176,6 +177,45 @@ namespace Gpu {
     {
         return makeExecutionConfig<MT>(box.numPts());
     }
+
+    struct ExecConfig
+    {
+        Long ntotalthreads;
+        int nblocks;
+    };
```

Comment:
```suggestion
    struct ExecConfig
    {
        Long start_idx;
        int nblocks;
    };
```
start_idx is the same as ndone just renamed

Reply:
Okay, I see what you mean.